### ISSUE TITLE
custom path

### ISSUE THREAD

**Issue by ruchitinfushion** [mottosso/be#47]
Where to define custom path for be.yaml, inventory.yaml & templates.yaml ?

**Comment by mottosso** [comment]
I don't think so, and I'm not sure how it would work. With `be`, the project configuration and folder must be one and the same.

**Example**

```
/projects/hulk/be.yaml
/projects/spiderman/be.yaml
```

What other folder would you like to store those files in?

**Comment by mottosso** [comment]
> Basically i want to hide this files from normal user

That makes sense.

Would be happy to see a PR for adding a `.` to the file names, that would hide them on Linux, and  then we could set file attributes for hiding them on Windows too.

**Comment by ruchitinfushion** [comment]
> Would be happy to see a PR for adding a `.` to the file names, that would hide them on Linux
yeah, good idea
